pkg/ts/catalog/catalog_generator.go

```diff
@@ -192,6 +192,8 @@ var chartCatalog = []ChartSection{
 	},
 }
 
+var catalogGenerated = false
```

Comment:
Why not create the catalog as an `func init() {...}` step when the binary starts up? That removes the awkwardness of the bool. But I don't know if all of the right data is available at startup. If not, this works.

Reply:
My thought here was simply that most people will never access this struct, so no need to put the fully populated tree in memory.

Reply:
OK, that's fine.